packages/image_picker/image_picker_android/example/android/app/build.gradle

```diff
@@ -0,0 +1,67 @@
+def localProperties = new Properties()
```

Comment:
This seems copy and pasted across examples, can we share this easily?

Reply:
I don't think there's any reasonable way for us to share native app boilerplate across examples. Especially since over time the dependencies (at the end) could reasonably diverge.

And while this is a duplicate in practice, it's also almost identical to what `flutter create` would have made for us if we'd made the new example from scratch instead.